### PR TITLE
Beat borgs with tools, #6727 done properly

### PR DESCRIPTION
**Pull request by Kegdo**
## About The Pull Request

I hate robot.dm I HATE robot.dm

## Why It's Good For The Game
Fixes not being able to harm borgs with certain tools

## Changelog
:cl:
fix: Fixed not being able to harm borgs with certain tools
/:cl:

### CODE REVIEW

**Comment by Kegdo** [comment]
@hyperioo here

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -536,117 +536,118 @@
...
+								chosen_tool.attackby(I,user)
+							else
+								to_chat(user, SPAN_WARNING("[src] has no modifiable tools."))
+						if("Radio")
+							if(!radio)
+								to_chat(user, SPAN_WARNING("Unable to locate a radio."))
+							if(I.use_tool(user, src, WORKTIME_FAST, tool_type, FAILCHANCE_NORMAL, required_stat = STAT_MEC))
+								radio.attackby(I,user)//Push it to the radio to let it handle everything
+								updateicon()
+				return
```
Comment (hyperioo) [inline code/modules/mob/living/silicon/robot/robot.dm:651]:
```suggestion


		if(ABORT_CHECK)
			return
```

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -536,118 +536,121 @@
...
 
 		if(ABORT_CHECK)
 			return
```
Comment (hyperioo) [inline code/modules/mob/living/silicon/robot/robot.dm:653]:
```suggestion
			return

```

**Review by hyperioo — APPROVED** [review]
Should be good now.
Integration tests failing because of
```
## ERROR: There are no map_sectors on the helm control console's z.
[08:32:29] Runtime in code/modules/maps/tg/reader.dm,94: Coords before model definition in DMM
[08:32:29] Runtime in code/modules/maps/tg/reader.dm,94: Coords before model definition in DMM
```
Should be fixed with #6701

**Comment by Kegdo** [comment]
Pain. Suffering. Agony, even.

**Pull request merged**